DEVELOPERS_GUIDE.md

```diff
@@ -276,7 +261,7 @@ To run the client, we use Angular CLI, as the client is an Angular app. All we h
 
 ```
 $ cd client
-$ ng serve --env=YOUR_ENV_NAME
+$ ng serve --env=personal
```

Comment:
See above why this is changed to `personal`.

Reply:
👍